pygmt/tests/test_colorbar.py

```diff
@@ -63,6 +22,6 @@ def test_colorbar_shading_list():
     Create colorbar and set shading by passing the high/low values as a list.
     """
     fig = Figure()
-    fig.basemap(region=[0, 10, 0, 10], projection="X15c", frame="a")
+    fig.basemap(region=[0, 10, 0, 10], projection="X8c", frame="a")
```

Comment:
Can make the y-axis smaller maybe?

```suggestion
    fig.basemap(region=[0, 10, 0, 2], projection="X8c", frame="a")
```